app/modules/ads/module.js

```diff
@@ -9,12 +10,17 @@ const SLOT_NAME_MAP = {
 	BOTTOM_LEADERBOARD: 'bottom-leaderboard'
 };
 
+const MOAT_JWPLAYER_PLUGIN_URL = 'https://z.moatads.com/jwplayerplugin0938452/moatplugin.js';
+
 class Ads {
 	constructor() {
 		this.engine = null;
 		this.events = null;
 		this.isLoaded = false;
 		this.onReadyCallbacks = [];
+		this.jwPlayerMoat = {
+			loadTrackingPlugin: () => window.M.loadScript(MOAT_JWPLAYER_PLUGIN_URL, true)
```

Comment:
What do you think about moving this method to video-ads? We can leave jwPlayerMoat object in ads module for now, but maybe it would be a bit cleaner to have MOAT loading in video module